scripts/run_performance_job.py

```diff
@@ -568,7 +568,7 @@ def run_performance_job(args: RunPerformanceJobArgs):
     if not args.internal and not args.performance_repo_ci:
         ci_setup_arguments.not_in_lab = True
 
-    if mono_dotnet is not None:
+    if mono_dotnet is not None and not mono_aot:
```

Comment:
Not sure if there is a case where we could have both mono_dotnet and mono_aot so this may not strictly be necessary, but it is probably good to still verify that it is not mono_aot.

Reply:
Yeah it's not possible, but just porting the exact same code from the previous script